dev/tasks/linux-packages/debian/control

```diff
@@ -24,6 +25,7 @@ Build-Depends:
   libsnappy-dev,
   libssl-dev,
   libzstd-dev,
+  meson,
```

Comment:
We can't add this because Meson provided by Debian and Ubuntu is old.
We need to install newer Meson by hand: https://github.com/apache/arrow/blob/master/dev/tasks/linux-packages/apt/debian-stretch-arm64/Dockerfile#L74-L75

Reply:
State of Meson:

- Buster has 0.49.2
- Stretch has 0.49.0
- Bionic has 0.45

Which version is required?

Reply:
My build under buster went fine.

Reply:
0.50.0 or later.

It's failed with 0.49.2:

```text
gandiva-glib/meson.build:50:0: ERROR:  Tried to create target "enums.c", but a target of that name already exists.
```

I think that you didn't enable Gandiva.

Reply:
Not enabled in previous build. Now doing build with meson 0.51.1 to check.

Reply:
> We can't require clang-7 package because there isn't clang-7 package on Debian stretch arm64.

LLVM 7 was backported to 'stretch' in August 2019 so we can enable it now.

Reply:
dropped meson dependency in newer patch